src/mgmt/rpc.h

```diff
@@ -130,6 +140,26 @@ CRT_RPC_DECLARE(mgmt_pool_find, DAOS_ISEQ_MGMT_POOL_FIND,
 
 #define MGMT_POOL_FIND_DUMMY_LABEL "NO LABEL, FINDING BY UUID"
 
+#define DAOS_SEQ_MGMT_POOL_LIST_POOL /* listed pool */           \
+	((uuid_t)		(plp_uuid)		CRT_VAR) \
+	((d_string_t)		(plp_label)		CRT_VAR) \
+	((d_rank_t)		(plp_svc_ldr)		CRT_VAR) \
+	((d_rank_list_t)	(plp_svc_list)		CRT_PTR)
```

Comment:
Do we need a uint32_t padding field between plp_svc_ldr and plp_svc_list?

Reply:
Done.